recipe/build.sh

```diff
@@ -24,6 +24,9 @@ cmake -G Ninja \
     -DLLVM_LIT_ARGS=-v \
     -DLLVM_CMAKE_DIR=$PREFIX/lib/cmake/llvm \
     -DLLVM_DIR=$PREFIX/lib/cmake/llvm \
+    -DLLVM_DEFAULT_TARGET_TRIPLE=wasm32-unknown-emscripten \
+    -DLLVM_TARGETS_TO_BUILD=WebAssembly \
+    -DLLVM_ENABLE_PROJECTS="clang;flang;mlir" \
```

Comment:
To understand, you want to build a compiler that runs on linux and compiles for emscripten? Not sure if our clang already supports that target; in any case, building clang/flang/mlir in the same feedstock will almost certainly time out.

We'll need to come up with a larger-scale plan for how we want to support wasm/emscripten in conda-forge, but as an experiment, I could imagine pushing this into a separate label `conda-forge/label/llvm_<...>`

Reply:
>  you want to build a compiler that runs on linux and compiles for emscripten?

Exactly.

I don't think it's necessary to rebuild clang. Emscripten uses clang internally.

For pushing to a new label, could I make the changes in this PR or should that be separate?